tests/scalability/README.md

```diff
@@ -94,3 +94,39 @@ From here we get the ddocs update sequences. Stage the branch which is associate
 Once the medic script completes we begin setting up for jmeter. The `start_jmeter_ec2.sh` script launches a new ec2 instance with the `user-data` script at `cht-core/tests/scalability/run_suite.sh`. 
 
 The script clones cht-core and navigates to the scalability dir. Then installs java, nodejs, jmeter, and its plugins. Executes the scalability suite which is defined in the `cht-core/tests/scalability/sync.jmx`, and finally it uploads the results to an s3 bucket. 
+
+
+
+## Distributed Workflow
+
+![Diagram of our current setup.](jmeter_distributed.png)
+
+Our current setup has 1 ec2 instance acting as the JMeter Orchestrator with 2 worker nodes. This can generate 1400 virtual users. Against our Application environment under test. 
+
+### Setup
+
+1. Launch an ec2 instance c5.2xlarge(Orchestrator)
+1. Launch 2 ec2 instances of c5.4xlarge(Worker)
+1. SSH into each of these machines. 
+1. Copy `tests/scalability/run_suite.sh` to the user directory. From this branch.
+1. Execute the `run_suite.sh`. This clones cht-core, installs JAVA, installs Node 14, runs npm install, installs JMeter, and its plugins.
```

Comment:
Do we need this? `chmod +x run_suite.sh` I thought git stored the execute permission of the file and when you scp it executed fine.

Reply:
Yes, git does! However, this script is what copies the repo down, so we had to get it otherwise. I used `wget` which doesn't of course preserve the execute bit.

Reply:
good point!